frontend/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "npg-longue-vue",
+  "version": "0.0.0",
+  "engines": {
+    "node": ">=16.16.0",
+    "npm": ">=8.11.0"
+  },
+  "scripts": {
+    "dev": "vite --port 3000",
+    "build": "vite build",
```

Comment:
What does this build produce (I assume static files)? I'm interested in knowing a good way for serving the app

Reply:
It builds a dist folder containing a very dull index.html that loads the app from a set of minified .js files under assets. If vite is anything like other build systems, it would be possible to tell it where to put that build target: https://vitejs.dev/guide/build.html#public-base-path

Reply:
> https://vitejs.dev/guide/build.html#public-base-path

I've been using that, though for resolving URLs in the app (e.g. mounting the app at `/some/path/index.html` rather than at the root).
`vite build --outDir=path` would change from the default `dist` if needed